nb/api/on.md

```diff
@@ -0,0 +1,39 @@
+### on()
```

Comment:
I'm not sure I knew about this and it's fantastic!

Reply:
Glad to hear it. :)